onnxruntime/core/optimizer/matmul_bn_fusion.cc

```diff
@@ -165,6 +181,8 @@ Status MatmulBNFusion::Apply(Graph& graph, Node& matmul_node, RewriteRuleEffect&
       bias_tensor->dims_size() != 1 ||
       mean_tensor->dims_size() != 1 ||
       var_tensor->dims_size() != 1 ||
+      // matmul_a_tensor->dims_size() != 2 ||
```

Comment:
nit: This comment line is confusing since matmul_a_tensor is not declared.

Reply:
removed